catalog/app/components/Markdown/Markdown.tsx

```diff
@@ -113,114 +112,86 @@ const highlight = (str: string, lang: string) => {
 interface RemarkableWithUtils extends Remarkable.Remarkable {
   // NOTE: Remarkable.Remarkable doesn't export utils
   utils: {
-    escapeHtml: (str: string) => string
-    replaceEntities: (str: string) => string
     unescapeMd: (str: string) => string
   }
 }
 
-const escape = R.pipe(
-  (Remarkable as unknown as RemarkableWithUtils).utils.replaceEntities,
-  (Remarkable as unknown as RemarkableWithUtils).utils.escapeHtml,
-)
+const { unescapeMd } = (Remarkable as unknown as RemarkableWithUtils).utils
 
-/**
- * A Markdown (Remarkable) plugin. Takes a Remarkable instance and adjusts it.
- *
- * @typedef {function} MarkdownPlugin
- *
- * @param {Object} md Remarkable instance.
- */
+const checkboxHandler = (md: Remarkable.Remarkable) => {
+  md.inline.ruler.push('tasklist', parseTasklist, {})
+  md.renderer.rules.tasklist = (tokens, idx) =>
+    (tokens[idx] as CheckboxContentToken).checked ? '☑' : '☐'
+}
 
-/**
- * Create a plugin for remarkable that does custom processing of image tags.
- *
- * @param {Object} options
- * @param {bool} options.disable
- *   Don't show images, render them as they are in markdown contents (escaped).
- * @param {function} options.process
- *   Function that takes an image object ({ alt, src, title }) and returns a
- *   (possibly modified) image object.
- *
- * @returns {MarkdownPlugin}
- */
-const imageHandler =
-  ({ disable = false, process = R.identity }) =>
-  (md: Remarkable.Remarkable) => {
-    // eslint-disable-next-line no-param-reassign
-    md.renderer.rules.image = (tokens, idx) => {
-      const t = process(tokens[idx])
+type AttributeProcessor = (attr: string) => string
 
-      if (disable) {
-        const alt = t.alt ? escape(t.alt) : ''
-        const src = escape(t.src)
-        const title = t.title ? ` "${escape(t.title)}"` : ''
-        return `<span>![${alt}](${src}${title})</span>`
-      }
+function handleImage(process: AttributeProcessor, element: Element): Element {
+  const attributeValue = element.getAttribute('src')
+  if (!attributeValue) return element
+  const result = process(attributeValue)
+  element.setAttribute('src', result)
 
-      const src = (Remarkable as unknown as RemarkableWithUtils).utils.escapeHtml(t.src)
-      const alt = t.alt
-        ? escape((Remarkable as unknown as RemarkableWithUtils).utils.unescapeMd(t.alt))
-        : ''
-      const title = t.title ? ` title="${escape(t.title)}"` : ''
-      return `<img src="${src}" alt="${alt}"${title} />`
-    }
+  const alt = element.getAttribute('alt')
+  if (alt) {
+    element.setAttribute('alt', unescapeMd(alt))
+  }
+  const title = element.getAttribute('title')
+  if (title) {
+    element.setAttribute('title', title)
```

Comment:
what's the purpose of getting the attribute and then setting it to the same value?

Reply:
It was leftover, when there was an escaping
I suppose, now, we don't need an escaping because we don't use strings for creating HTML elements